src/routes/jira/server/jira-ghe-servers-router.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import { JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
+import { csrfMiddleware } from "middleware/csrf-middleware";
+import { JiraGheServers } from "routes/jira/server/jira-ghe-servers";
+
+export const JiraGheServersRouter = Router();
+
```

Comment:
Worth moving these to a .use i reckon
 ```JiraGheServersRouter.use(csrfMiddleware)```
 ```JiraGheServersRouter.use(JiraJwtTokenMiddleware)```

Reply:
For this new route, I'm simply following the structure of the other existing routes. So, I'm not sure which ones the correct way.

Reply:
The router file has been deleted and used `get` in the JiraRouter.